### PR TITLE
Define unacceptable decisions

### PR DESCRIPTION
**Pull request by anthonygc**
## Description

Changed the unacceptable conduct for decisions.

### Type of Change

- [ ] Feature
- [ ] Bug fix
- [x ] Documentation

## Contributors

- @anthonygc

## Reminder

All GitHub Actions checks must be in a passing state in this branch or fork before this pull request can be merged.

### CODE REVIEW

**Review by Jacob-Allebach — DISMISSED** [review]
I think these points seem good!

File: conduct.md [hunk 1]
```diff
@@ -96,9 +96,9 @@ TODO: Write an introductory paragraph.
 
 #### Unacceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Students should not talk over others or prevent others from submitting their ideas.
+- Students should not commit and push code that falls outside of the group's initial intent for a project.
+- Students should not avoid reviewing documentation after a decision for the project has been made.
```
Comment (PaigeCD) [inline conduct.md]: I think line 101 could be a bit clearer for me. By "reviewing documentation" do you mean "updating"?

**Review by PaigeCD — CHANGES_REQUESTED** [review]
Overall very good points! I just have one comment for you to look at.

File: conduct.md (hunk 1 above)
Comment (Ochirsaikhan) [reply conduct.md]: I agree. Other than the last point, it seems good to me 👌

Comment (DlHolland) [reply conduct.md]: Makes sense to me!

Comment (anthonygc) [reply conduct.md]: Appreciate it guys! Thanks for the suggestions!

File: conduct.md [hunk 2]
```diff
@@ -96,9 +96,9 @@ TODO: Write an introductory paragraph.
 
 #### Unacceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Students should not talk over others or prevent others from submitting their ideas.
+- Students should not commit and push code that falls outside of the group's initial intent for a project.
+- Students should not update documentation or code without consulting the group about it.
```
Comment (mariakimheinert) [inline conduct.md]: Is it possible that this statement is too restrictive? Could it be better to say that students should make sure to change documentation or code in ways that are consistent with the decisions made by the team?

**Review by mariakimheinert — CHANGES_REQUESTED** [review]
Hi @anthonygc, thanks for these changes! I've made one suggestion below. Please let me know if you have any questions about my comment!

**Review by mariakimheinert — APPROVED** [review]
@anthonygc, these changes look great! Thanks for all your hard work. :shipit:

**Review by PaigeCD — APPROVED** [review]

**Pull request merged**